src/billing/components/BillingContactInfo.tsx

```diff
@@ -8,31 +8,20 @@ import {
 } from '@influxdata/clockface'
 
 // Components
+import {BillingContext} from 'src/billing/context/billing'
 import BillingContactForm from 'src/billing/components/Checkout/BillingContactForm'
 import BillingContactDisplay from 'src/billing/components/BillingContactDisplay'
 
-// Utils
-import {useBilling} from 'src/billing/components/BillingPage'
-import {getBillingInfo} from 'src/billing/thunks'
-
 const BillingContactInfo: FC = () => {
-  const [
-    {
-      billingInfo: {contact},
-    },
-    dispatch,
-  ] = useBilling()
+  const {
+    billingInfo: {contact},
+  } = useContext(BillingContext)
 
   // Contact is created during signup but city (required) is not collected then
   const isFirstContactSaved = contact && contact.city
 
   const [isEditing, setIsEditing] = useState(!isFirstContactSaved)
 
-  const handleSubmitEditForm = () => {
-    setIsEditing(false)
-    getBillingInfo(dispatch)
```

Comment:
it makes no sense for us to `GET` the billing info AGAIN after editing the form since the API should return the results when we run the `PUT`